views/community.ejs

```diff
@@ -0,0 +1,269 @@
+        padding: 0 10px;
+    }
+
+    .myTable td{
+        border-top: 1px solid #D3D3D3;
+        padding: 0 10px;
+    }
+
+
+    .myTable tr:hover {
+    	background-color:#c5cae9;
+    }
+
+    .myTable img {
+        height:50px;
+        padding: 5px 0;
+        object-fit: contain;
+    }
+
+    .myTable p {
+        background-color: transparent !important;
+        margin: 0;
+        display: flex;
+        height: 60px;
+        justify-content: center;
+        flex-direction: column;
+    }
+
+    .bio {
+        width: 45%;
+    }
+
+    .cell {
+        height: 60px;
+        display: block;
+        overflow: auto;
+/*        display: flex;
+        flex-direction: column;*/
+    }
+
+    ::-webkit-scrollbar {
+    width: 0px;  /* remove scrollbar space */
+    background: transparent;  /* optional: just make scrollbar invisible */
+    }
+
+    h4 {
+        margin: 0 0 0 5%;
+        padding: 0 10px;
+        color: #3f51b5;
+        font-weight: 500;
+    }
+
+    button {
+        margin: 0 0 0 5%;
+    }
+
+    /*hide games played*/
+    .games{
+        display:none;
+        width:0;
+        height:0;
+        opacity:0;
+        visibility: collapse;
+    } 
+
+@media screen and (max-width: 600px) {
+    .myTable {
+        font-size: 11px;
+    }
+    .myTable td th {
+        word-wrap: break-word;
+        padding: 0;
+    }
+
+    .myTable img {
+        height:40px;
+        padding: 0;
+    }
+
+    .cell {
+        height: 60px;
+        display: block;
+        overflow: auto;
+    }
+
+    ::-webkit-scrollbar {
+    width: 0px;  /* remove scrollbar space */
+    background: transparent;  /* optional: just make scrollbar invisible */
+    }
+
+    /*hide nat*/
+    .nat{
+        display:none;
+        width:0;
+        height:0;
+        opacity:0;
+        visibility: collapse;
+    } 
+}
+
+
+</style>
+
+<div class="myContainer">
+	<h3 class="header">View our players from all over the world!</h3>
+	<div class="row">
+        <table class="myTable">
+            <h4>Moderators</h4>
+            <tr>
+                <th>Username</th>
+                <th>Res Avatar</th>
+                <th>Spy Avatar</th>
+                <th class="nat">Nationality</th>
+                <th class="bio">Biography</th>
+                <th class="games">#Games</th>
+            </tr>
+            
+            <% mods.forEach(function(mod){ %>
+            <tr>
+                <td><%= mod.username %></td>
+                <%if (mod.avatarImgRes === null) { %>
+                    <td><img src="https://www.proavalon.com/avatars/base-res.png"></td>
+                <% } else { %>
+                    <td><img src="<%= mod.avatarImgRes %>"></td>
+                <% } %>
+                <%if (mod.avatarImgSpy === null) { %>
+                    <td><img src="https://www.proavalon.com/avatars/base-spy.png"></td>                
+                <% } else { %>
+                    <td><img src="<%= user.avatarImgSpy %>"></td>
```

Comment:
This should probably say `mod.avatarImgSpy` instead of `user.avatarImgSpy`...

Reply:
Yeah you're right, I fixed it